pip/wheel.py

```diff
@@ -668,19 +668,24 @@ def _build_one(self, req, output_dir):
                     logger.info('Stored in directory: %s', output_dir)
                     return wheel_path
                 except:
-                    return None
+                    pass
```

Comment:
Even if there was no logging previously I think a logger.debug of the exception could be helpful.